### DISCUSSION TITLE
from char array or string to json

### DISCUSSION THREAD

**Discussion by Leczok12** [nlohmann/json#3818] [General]
I am trying to convert this "{\r\n\t\"eventname\": \"OnDeathAssist\",\r\n\t\"other\": \"Turret_T2_R_03_A\",\r\n\t\"otherTeam\": \"Chaos\",\r\n\t\"source\": \"NL luzak54\",\r\n\t\"sourceID\": \"4\",\r\n\t\"sourceTeam\": \"Order\"\r\n}\n" to json by json::parser.
Every time i tryed json are empyt or i have a error. I saw in documentation how propably slove this by add in end a _json bad it's not working with variables.

**Comment by falbrechtskirchinger** [comment]
Please provide a [minimal reproducible example](https://en.wikipedia.org/wiki/Minimal_reproducible_example).
